pkg/config/config.go

```diff
@@ -82,11 +82,19 @@ func getKernelVersion(c Client) float32 {
 	}
 	// per https://github.com/google/cadvisor/blob/master/machine/info.go#L164
 	kv := utsname.Release[:bytes.IndexByte(utsname.Release[:], 0)]
+	// for something like 4.18.10-24-generic
 	val, err := strconv.ParseFloat(string(kv[:4]), 32)
 	if err == nil {
 		return float32(val)
+	} else {
```

Comment:
how about splitting the string by `.`, and taking the first two substrings, and recreating the new kernel string?

Reply:
+1